.github/workflows/pr-chart-sync.yaml

```
@@ -0,0 +1,157 @@
+      contents: write
+      pull-requests: write
+    env:
+      BROWSER_DOWNLOAD_URL: ${{ github.event.inputs.browser_download_url }}
+      CHART_NAME: ${{ github.event.inputs.chart }}
+      CHART_VERSION: ${{ github.event.inputs.version }}
+      COMPONENT_REPO: ${{ github.event.inputs.component_repo }}
+      BRANCH_NAME: "sync/${{ github.event.inputs.chart }}-${{ github.event.inputs.version }}"
+
+    steps:
+      - name: Set up Go 1.21.x
+        uses: actions/setup-go@v4
```

actionlint (1.7.12)

[error] the runner of "actions/setup-go@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)